src/database/migrations/20190729080444-create-commentlikes.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable(
+      'CommentLike',
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER
+        },
+        commentId: {
+          type: Sequelize.INTEGER,
+          onDelete: 'CASCADE',
+          references: {
+            model: 'Comments',
+            key: 'id',
+            as: 'commentId'
+          }
+        },
+        userId: {
+          type: Sequelize.INTEGER,
+          onDelete: 'CASCADE',
+          references: {
+            model: 'Users',
+            key: 'id',
+            as: 'userId'
+          }
+        },
+        liked: {
```

Comment:
A user should only be able to like a comment and NOT unlike. Hence if the user has liked the comment before, they should be able to remove such like on the comment. I think there is no need or this column in that case. 

This table should only reflect list of people that have liked a specific article and when someone removes there like? then the row should be removed from this table